wiki/Bazel-Setup-Instructions-for-Linux.md

```diff
@@ -0,0 +1,67 @@
+## Instructions
+
+**The steps to install Bazel on Linux are:**
+1. Install Bazel 
+2. Install the OpenJDK 8
+3. Install Python 2 and make sure it is active in your environment
+4. Set the Android SDK
```

Comment:
Set up the ANDROID_HOME environment variable (ditto for header 4 below)

Reply:
Done